R/ml_fit_dss.R

```diff
@@ -6,11 +6,11 @@
 #' @param ginv Function that computes the Moore-Penrose pseudoinverse.
 #' @references Deville, J.-C. and \enc{Särndal}{Saerndal}, C.-E. (1992)
 #' Calibration estimators in survey sampling. *Journal of the American
-#' Statistical Association*, **87 ** (418), 376--382.
+#' Statistical Association*, **87** (418), 376--382.
```

Comment:
I remember this was on purpose, otherwise there is too little space to the subsequent number.

Reply:
This seems ok now?

![image](https://user-images.githubusercontent.com/17020181/122194300-a7e51c80-ced8-11eb-9859-8bbf352a85cd.png)